todo/tests/unit/views/test_task.py

```
@@ -0,0 +1,59 @@
+from unittest import TestCase
+from rest_framework.test import APIRequestFactory
+from rest_framework import status
+from rest_framework.exceptions import ValidationError
```

Ruff (0.8.2)

`rest_framework.exceptions.ValidationError` imported but unused

Remove unused import: `rest_framework.exceptions.ValidationError`

(F401)